continuous_integration/environment-3.10-dev.yaml

```diff
@@ -13,7 +13,7 @@ dependencies:
 - mlflow
 - mock
 - nest-asyncio
-- pandas>=1.1.2
+- pandas>=1.4.0
```

Comment:
Just leaving a note that we'll need to bump our Pandas version to get around a `setuptools` version ceiling added in https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/204 that would otherwise cause conflicts